cves/kernel/CVE-2020-2732.yml

```diff
@@ -344,8 +362,10 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    It was signed off by Oliver Upton <oupton@google.com> and reviewed by Vitaly 
```

Comment:
I would also include Paolo Bonzini <pbonzini@redhat.com> who was involved in signing off on the three commits